src/brokers/broker-pods/components/BrokerPodsBreadcrumb/BrokerPodsBreadcrumb.test.tsx

```diff
@@ -52,4 +89,65 @@ describe('BrokerPodsBreadcrumb', () => {
     fireEvent.click(toggle);
     expect(toggle).toHaveAttribute('aria-expanded', 'true');
   });
+
+  it('should navigate back to PodsList page when click on Apply button in both form and yaml editor', async () => {
+    const comp = render(
+      <MemoryRouter>
+        <BrokerPodsBreadcrumb name={name} namespace={namespace} />
+      </MemoryRouter>,
+    );
+    await waitForI18n(comp);
+
+    fireEvent.click(screen.getByTestId('broker-toggle-kebab'));
+    fireEvent.click(screen.getByText('edit_broker'));
+
+    expect(navigate).toHaveBeenCalledWith(
+      `/k8s/ns/${namespace}/edit-broker/${name}?returnUrl=${encodeURIComponent(
+        window.location.pathname,
+      )}`,
+    );
+  });
```

Comment:
Same, this is testing the behavior of the apply button, why is this test in the breadcrumb test file?